client/src/store/new-request-module.ts

```diff
@@ -1836,7 +1837,7 @@ export class NewRequestModule extends VuexModule {
         let fields = Object.keys(canadaPostFieldsMapping)
         for (let field of fields) {
           if (addressData[field]) {
-            let value = addressData[field].toUpperCase()
+            let value = addressData[field]
```

Comment:
There's a ticket for this issue. Please link it to this PR.

Reply:
/bcgov/entity/5837

Reply:
I will inform Tracey